common/script/ops/reset.js

```diff
@@ -1,35 +1,22 @@
-import _ from 'lodash';
+import resetGear from './resetGear';
+import i18n from '../i18n';
 
-module.exports = function(user, req, cb) {
-  var gear;
-  user.habits = [];
-  user.dailys = [];
-  user.todos = [];
-  user.rewards = [];
+module.exports = function reset (user) {
+  user.tasksOrder.habits = [];
```

Comment:
these 4 lines don't make sense as we remove the necessary ids from tasksOrder using `removeNonChallengeTasks ` and anyway not all ids should be removed. 

I'd just keep `removeNonChallengeTasks` and add a comment here to do the same on the client side